tests/client_test.c

```diff
@@ -1034,7 +1034,7 @@ static void test_allocator_injection(void) {
     valkeyResetAllocators();
 }
 
-#define VALKEY_BAD_DOMAIN "idontexist-noreally.com"
+#define VALKEY_BAD_DOMAIN "nonexistent.example.com"
```

Comment:
https://www.rfc-editor.org/rfc/rfc2606.html

Reply:
I'm fine with this change. The reason i did it the first way was because someone actually registered `idontexist.com` so the tests were failing.